### PR TITLE
C360115 Verify that "Start bulk edit" option is not available after u…

### PR DESCRIPTION
**Pull request by KarapetyanSofia**
[C360115](https://foliotest.testrail.io/index.php?/cases/view/360115)

[Allure Report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5572/allure)

image

### CODE REVIEW

File: cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js [hunk 1]
```diff
@@ -683,7 +683,17 @@ export default {
       cy.expect(Button('Download errors (CSV)').exists());
     }
   },
-
+  verifyActionsAfterErrors(errors = true) {
+    cy.do(actions.click());
+    cy.expect([
+      Button('Download matched records (CSV)').absent(),
+      Button('Start bulk edit').absent(),
+      DropdownMenu().find(HTML('Show columns')).exists(),
```
Comment (manowarqqq) [inline cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js:691]: Please, check  "Show columns" subsection is disabled.

Comment (KarapetyanSofia) [reply cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js:691]: I already fixed it!

File: cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js [hunk 2]
```diff
@@ -683,7 +683,17 @@ export default {
       cy.expect(Button('Download errors (CSV)').exists());
     }
   },
-
+  verifyActionsAfterErrors(errors = true) {
```
Comment (nayimovag) [inline cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js:686]: this is a redundant method, there are several methods that check these separately, please use those

File: cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js [hunk 3]
```diff
@@ -711,7 +721,47 @@ export default {
       DropdownMenu().find(Checkbox('Custom fields')).has({ checked: false }),
     ]);
   },
-
+  verifyShowColumnsSubsection() {
+    cy.expect([
+      DropdownMenu().find(Checkbox('Holdings ID')).has({ disabled: true }),
```
Comment (nayimovag) [inline cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js:726]: just verify that Checkbox with filter disabled: false is absent in the dropdown

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-option-is-not-available.cy.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  instanceId: '',
+  holdingUUID: '',
+  holdingHRID: '',
+};
+
+const invalidHoldingUUIDsFileName = `InvalidHoldingUUIDs_${getRandomPostfix()}.csv`;
+let holdingUUID;
+
+describe('bulk-edit', () => {
+  describe('Holdings', () => {
```
Comment (nayimovag) [inline cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-option-is-not-available.cy.js:24]: make sure describe blocks' texts match with other tests in the same folder

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-option-is-not-available.cy.js [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+        item.instanceId = InventoryInstances.createInstanceViaApi(
+          item.instanceName,
+          item.itemBarcode,
+        );
+        cy.getHoldings({
+          limit: 1,
+          query: `"instanceId"="${item.instanceId}"`,
+        }).then((holdings) => {
+          holdingUUID = holdings[0].hrid;
```
Comment (nayimovag) [inline cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-option-is-not-available.cy.js:41]: can you use uuid() or getRandomPostfix() instead

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-option-is-not-available.cy.js [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+import FileManager from '../../../support/utils/fileManager';
+import Users from '../../../support/fragments/users/users';
+
+let user;
+const item = {
+  instanceName: `testBulkEdit_${getRandomPostfix()}`,
+  itemBarcode: getRandomPostfix(),
+  instanceId: '',
+  holdingUUID: '',
+  holdingHRID: '',
```
Comment (nayimovag) [inline cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-option-is-not-available.cy.js:17]: do you need these instantiated here? instanceId: '',
  holdingUUID: '',
  holdingHRID: '',

Comment (KarapetyanSofia) [reply cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-option-is-not-available.cy.js:17]: instanceId and  holdingUUID I`ve used, only holdingHRID doesnt need!

File: cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js (hunk 3 above)
Comment (KarapetyanSofia) [reply cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js]: Done!

File: cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js [hunk 7]
```diff
@@ -711,7 +711,13 @@ export default {
       DropdownMenu().find(Checkbox('Custom fields')).has({ checked: false }),
     ]);
   },
-
+  verifyShowColumnsSubsection() {
```
Comment (manowarqqq) [inline cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js:714]: Unclear method's name

**Review by manowarqqq — APPROVED** [review]

**Review by nayimovag — APPROVED** [review]

**Pull request merged**